bftengine/tests/simpleKVBCTests/simpleKVBCTests.cpp

```diff
@@ -114,7 +110,7 @@ namespace BasicRandomTests
 		struct SimpleConditionalWriteHeader
 		{
 			SimpleRequestHeader h; // TODO: this is ugly ....
-			BlockId readVerion;
```

Comment:
We do a duplicate job...

Reply:
Yep. We'll discuss tomorrow as mentioned offline.